services/src/kilda-core/kilda-model/src/main/java/org/openkilda/model/Flow.java

```diff
@@ -85,12 +85,24 @@
     @Property(name = "src_vlan")
     private int srcVlan;
 
+    @Property(name = "detect_src_lldp_connected_devices")
+    private boolean detectSrcLldpConnectedDevices;
```

Comment:
Please, consider wrapping the new properties into a dedicated class. This class will be mapped to a set of Flow fields in the storage.

Reply:
added converter